src/js/smooch.jsx

```diff
@@ -161,6 +164,14 @@ export class Smooch {
                 }
             });
         }).then((loginResponse) => {
+            observable.on('socket:connected', () => {
```

Comment:
Yeah I would prefer dispatching all of these events via faye. No half measures.

![image](https://cloud.githubusercontent.com/assets/2432382/13891055/4d214b1a-ed25-11e5-8aea-0af342219b3a.png)